test/integration.test.js

```diff
@@ -3,71 +3,69 @@
 'use strict'
 
 import { join } from 'path'
-import next from '../dist/server/next'
 import pkg from '../package.json'
+import next from '../dist/server/next'
+import {expectElement, setup, render, teardown} from 'next-test-helper'
 
+let app = null
 const dir = join(__dirname, 'fixtures', 'basic')
-const app = next({
-  dir,
-  dev: true,
-  staticMarkup: true,
-  quiet: true
-})
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000
 
 describe('integration tests', () => {
-  beforeAll(() => app.prepare())
-
-  afterAll(() => app.close())
+  beforeAll(async () => {
+    app = await setup(dir, next)
+  })
+  afterAll(async () => await teardown())
```

Comment:
👍 wasn't sure if Jest was smart enough to detect that the `teardown` was `async`/`await` I'll give this a try.